app/cdap/components/WranglerGrid/SelectColumnPanel/ColumnsList/types.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import {
+  IHeaderNamesList,
+  IDataQualityItem,
+} from 'components/WranglerGrid/SelectColumnPanel/types';
+
+export interface ISelectColumnsListProps {
+  transformationDataType: string[];
+  selectedColumnsCount: number;
+  columnsList: IHeaderNamesList[];
+  setSelectedColumns: React.Dispatch<React.SetStateAction<IHeaderNamesList[]>>;
+  dataQuality: IDataQualityItem[];
+  transformationName: string;
```

Comment:
Can this be any string or just the values inside of MULTI_SELECTION_COLUMN? If it a small list of known values, they should be listed here. 'oneValue' | 'anotherValue' - or include it directly as a list of values.

Reply:
transformationName will be the values of all the nested options present in the toolbar like -- join, swap, uppercase, lowercase, etc.

Reply:
Then it's better to limit this type to the known values similar to what Sean suggested rather than a generic `string` type.

Reply:
Hi @seanfeld and @mrahanjam the known values are more than 20 so it's better to have generalised string rather than including all of them here. Let us know otherwise